park_api/cities/Bonn.py

```diff
@@ -31,6 +33,19 @@ def parse_html(html):
             "free": int(h3[1].span.strong.get_text()),
             "address": lot.address,
             "total": lot.total,
+            "state": "unknown",
+            "id": lot.id,
+            "forecast": False
+         })
+        else:
+          name = h3[0].string
+          lot = geodata.lot(name)
+          lots.append({
+            "name": name,
+            "coords": lot.coords,
+            "free": 0,
+            "address": lot.address,
```

Comment:
this could be written more condensed and also use `is None` instead of `== None` to check if something is not set

```
free = 0
if h3[0].a is not None:
  free = int(h3[1].span.strong.get_text())
lots.append(
   #...
   "coords": lot.coords,
   "free": free,
   #...
)
```